### PR TITLE
Don't modify /etc/hosts to simulate a node being down

### PR DESCRIPTION
**Pull request by cawallin**
* Change topology to have an invalid IP address for a node that is
  down, rather than modifying /etc/hosts.
* Had to modify some test results

Testing: make clean test-all on both cluster and docker

### CODE REVIEW

File: tests/docker_cluster.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def __init__(self, master_host, slave_hosts,
     def all_hosts(self):
         return self.slaves + [self.master]
 
-    def all_internal_hosts(self):
+    def all_internal_hosts(self, stopped_host=None):
```
Comment (rschlussel) [inline tests/docker_cluster.py]: is stopped_host used?

Comment (cawallin) [inline tests/docker_cluster.py]: it's used in configurable_cluster; wanted the method signature to be the same

Comment (cawallin) [inline tests/docker_cluster.py]: It's not necessary. I removed it.

Comment (cawallin) [inline tests/docker_cluster.py]: oh, I remember why I had it. it's because you don't know which host is the stopped host and the name is wrong.

**Pull request merged**